docs/src/explanations/ports.md

```diff
@@ -63,5 +70,5 @@ class Test extends Module {
   }}
 }
 
-chisel3.Driver.execute(Array[String](), () => new Test)
-```
+//(new ChiselStage).execute(Array[String](), () => new Test)
```

Comment:
@seldridge @jackkoenig not quite sure how to translate this.

Reply:
Annoyingly, I'm pretty sure it's this:
```scala
(new ChiselStage).execute(args, ChiselGeneratorAnnotation(() => new Test) :: Nil)
```
https://scastie.scala-lang.org/5ih1KYjjRYKwQ7NJdjo3aw

I think we should add another `execute` method that does that wrapping for you. Thoughts @seldridge?

Reply:
hm, actually this one works for me now as uncommented. There is another one where I get a null pointer exception I will note in a moment...